inc/functions.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Core sitemap public functions.
+ *
+ * @package Core_Sitemaps
+ */
+
+/**
+ * Get a list of sitemaps.
+ *
+ * @return array $sitemaps A list of registered sitemap providers.
+ */
+function core_sitemaps_get_sitemaps() {
```

Comment:
This is the first of many public functions we'll want to create that returns data from the global `Core_Sitemaps` object so that people using sitemaps functionality won't have to pass objects around when working with sitemaps data.